hacks/generateOptimizeSidebars.md

```diff
@@ -6,79 +6,50 @@ With the Optimize docs living in their own docusaurus docs instance, we need to
-     - **`docsAndVersionUrlPrefix`**: This value will be stripped from all generated URLs, so that the `optimize_sidebars.js` file can prepend the correct version in only one place instead of every single link. This will help us avoid a large find/replace when versioning optimize docs.
+   You will need to generate definitions on 6 different pages per version:
 
-       Note that this setting only has an effect when generating .js instead of .json.
+   - A main docs page under the Components top-navigation category. Example: https://docs.camunda.io/docs/components/
+   - A main docs page under the APIs & Clients top-navigation category. Example: https://docs.camunda.io/docs/apis-tools/working-with-apis-tools/
+   - A main docs page under the Self-Managed top-navigation category. Example: https://docs.camunda.io/docs/self-managed/about-self-managed/
+   - An Optimize page under the Components top-navigation category. Example: https://docs.camunda.io/optimize/components/what-is-optimize/
+   - An Optimize page under the APIs & Clients top-navigation category. Example: https://docs.camunda.io/optimize/apis-tools/optimize-api/optimize-api-authorization/
+   - An Optimize page under the Self-Managed top-navigation category. Example: https://docs.camunda.io/optimize/self-managed/optimize-deployment/install-and-start/
 
-   - The last statement of the script copies the generated sidebars into your clipboard.
+   This is because the left sidebar only contains links to pages within the same top-navigation category, and there are three top-navigation categories that contain Optimize docs.
 
-4. Paste the results into the optimize_sidebars.js file for the "next" version, or the appropriate versioned sidebars.json file in optimize_versioned_sidebars
+   **What the script does**
 
-   - When generating .js, the results do not include the statement `module.exports = `, so leave that part in 😅.
+   1. Identifies the configuration settings for the script, based on the current URL.
+   2. Expands all sidebar items on the current page (so that they are available to crawl in the DOM).
+   3. Crawls the DOM for all sidebar items, and generates sidebar definitions.
+   4. Copies the generated sidebar definitions to your clipboard.
 
-   - Make sure the results get formatted, as the script generates an object that does not match prettier's desired format.
+   **Steps for each page**
 
-5. Replace/revert any Optimize links that exist within the optimize docs
+   1. Execute the script by running `await generateOptimizeSidebars()` in the console.
+   2. Paste the contents of your clipboard into the _opposite_ instance's corresponding version sidebars file than the page you visited to generate the definitions. If you visited an Optimize v3.11.0 page, the generated definitions should go in the main instance's v8.3 sidebars file. If you visited a main docs v8.2 page, the generated definitions should go in the Optimize instances v3.10.0 sidebars.
```

Comment:
This will get stale overtime, but we don't have a good table that shows the mapping of Optimize to C8 versions.

Reply:
we do have this file that we could link to from here - https://github.com/camunda/camunda-docs/blob/25fe27a8f594d33f8cd5e5bd578c041020720bf6/src/versions.js#L16-L37